docs/client-development/mandate.rst

```diff
@@ -0,0 +1,526 @@
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+            "source": "eherkenning",
+            "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+            "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+        }
+    }
+
+
+4. eHerkenning / With mandate (bewindvoering)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**eHerkenning - initiator**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
+        },
+        "authenticatieContext": {
+            "source": "eherkenning",
+            "levelOfAssurance": "urn:etoegang:core:assurance-class:loa2plus",
+            "representee": {
+                "identifierType": "bsn",
+                "identifier": "111222333"
+            },
+            "actingSubject": "4B75A0EA107B3D36C82FD675B5B78CC2F181B22E33D85F2D4A5DA63452EE3018@2D8FF1EF10279BC2643F376D89835151"
+            "mandate": {
+                "role": "bewindvoerder",
+                "services": [{
+                    "id": "urn:etoegang:DV:00000001002308836000:services:9113",
+                    "uuid": "34085d78-21aa-4481-a219-b28d7f3282fc"
+                }]
+            }
+        }
+    }
+
+
+**eHerkenning - belanghebbende**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "natuurlijk_persoon",
+        "roltype": "http://example.com/roltype-belanghebbende",
+        "roltoelichting": "Persoon waarover bewind gevoerd wordt",
+        "indicatieMachtiging": "machtiginggever",
+        "betrokkeneIdentificatie": {
+            "inpBsn": "111222333"
+        },
+        "authenticatieContext": null
+    }
+
+
+5. eHerkenning / With mandate (ketenmachtiging)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+**eHerkenning - initiator**
+
+.. code:: http
+
+    POST /zaken/api/v1/rollen HTTP/1.1
+    Content-Type: application/json
+
+    {
+        "zaak": "http://example.com",
+        "betrokkeneType": "vestiging",
+        "roltype": "http://example.com/roltype-initiator",
+        "roltoelichting": "Created zaak",
+        "contactpersoonRol": {
+            "naam": "acting subject name"
+        },
+        "indicatieMachtiging": "gemachtigde",
+        "betrokkeneIdentificatie": {
+            "kvkNummer": "12345678",
+            "vestigingsNummer": "123456789012"
```

Comment:
:warning: this is not possible with ketenmachtiging, it's KVK -> KVK only

... which makes this case unsupported, since you can't use vestiging with branch number, and nnp doesn't support KVK numbers

Reply:
Discussed with Sergei, and a warning about this in the documentation would be sufficient for now. 'Ketenmachtigingen' is considered out of scope for the requested change

Reply:
We've added `kvk_nummer` attribute to `NietNatuurlijkPersoon`, so now you can create a Rol with nnp and fill in its KVK nummer. 
Should I change this example to "nnp" and add a warning, that it's not supported for "vestiging"?


Also `vestigings_nummer` is not a required field for "Vestiging" rol, so it's possible to create a rol with type Vestiging and fill in only KVK nummer. But I don't know should we encourage such behavior in the documentation

Reply:
Ah I see, that's a nice quality of life improvement that I really hope VNG will accept :crossed_fingers: 

> Should I change this example to "nnp" and add a warning, that it's not supported for "vestiging"?

Honestly, if we can sneak in the KVKNr into the standard, I'm all for it!

> Also vestigings_nummer is not a required field for "Vestiging" rol, so it's possible to create a rol with type Vestiging and fill in only KVK nummer. But I don't know should we encourage such behavior in the documentation

I've no idea if this is deliberate in the standard or not - if we can track that down to a deliberate change, that would be good motivation to "endorse" it but I do admit it feels wrong.